options.go

```
@@ -82,8 +82,8 @@ type TransportOptions struct {
 
 // BenchmarkOptions are benchmark-specific options
 type BenchmarkOptions struct {
-	MaxRequests int           `short:"n" long:"max-requests" default:"1000000" description:"The maximum number of requests to make"`
-	MaxDuration time.Duration `short:"d" long:"max-duration" default:"0s" description:"The maximum amount of time to run the benchmark for"`
+	MaxRequests int           `short:"n" long:"max-requests" default:"0" description:"The maximum number of requests to make. Values <= 0 imply no limit."`
```

Good point, only 0 should be supported. Negative values should probably throw an error.